docs/release-info/migr-subquery-limit.md

```diff
@@ -0,0 +1,55 @@
+---
+id: migr-subquery-limit
+title: "Migration guide: Subquery limit"
+sidebar_label: Subquery limit
+---
+
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+-->
+
+:::info
+The byte-based subquery limit is an [experimental feature](../development/experimental.md) introduced in Druid 30.0.0.
+:::
+
+Druid now allows you to set a byte-based limit on subquery size, to prevent brokers from running out of memory when handling large subqueries. Druid uses subqueries as joins as well as in common table expressions, such as WITH.
+
+The byte-based subquery limit works in conjunction with Druid's row-based subquery limit.
```

Comment:
Byte-based supersedes the row-based limit. If for some reason (most likely a bug or an unsupported column type) we aren't able to materialize the subquery as bytes, the row based limit is used as a fallback. "conjunction" doesn't seem correct, since it implies that both are applied to a single query.

Reply:
replaced with 'overrides'